### PR TITLE
Corrections to Disable-ServerManagerStandardUserRemoting.md

### LINKED ISSUE #3325: Quality: ServerManager
### Prerequisites

- [X] **Existing Issue:**

Search the existing [quality issues][b1] for this repository. If there's an issue that
covers the quality area and articles you want to improve, pick a different quality area
or article set. Don't file a new issue for the same work. Subscribe, react, or comment
on that issue instead.

[b1]: https://github.com/orgs/MicrosoftDocs/projects/15/views/4

- [X] **Descriptive Title:**

Write the title for this issue as a short synopsis. If possible, provide context. For
example, "Quality: Command Syntax in Foo" instead of "Quality: Foo"

### Quality Areas

Formatting code samples, Formatting command syntax

### Article List

Disable-ServerManagerStandardUserRemoting.md

### PR DESCRIPTION
**Pull request by raynbowbrite**
# PR Summary

Formatting and grammatical corrections to documentation for Disable-ServerManagerStandardUserRemoting.md

- Fixes #3325

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

### CODE REVIEW

**Comment by raynbowbrite** [comment]
@microsoft-github-policy-service agree

File: docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md [hunk 1]
```diff
@@ -11,8 +11,10 @@ title: Disable-ServerManagerStandardUserRemoting
 # Disable-ServerManagerStandardUserRemoting
 
 ## SYNOPSIS
-Disables access for specified standard users to event, service, performance counter, and role and feature inventory data that is collected by Server Manager for a server.
-This cmdlet performs the opposite action, for specified users, of the Enable-ServerManagerStandardUserRemoting cmdlet.
+
+Disables access for specified standard users to event, service, performance counter, and role and
+feature inventory data that is collected by Server Manager for a server. This cmdlet performs the
+opposite action, for specified users, of the Enable-ServerManagerStandardUserRemoting cmdlet.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md:17]:
```suggestion
opposite action, for specified users, of the `Enable-ServerManagerStandardUserRemoting` cmdlet.
```

File: docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md [hunk 2]
```diff
@@ -21,40 +23,57 @@ Disable-ServerManagerStandardUserRemoting [-User] <String[]> [-Force] [-WhatIf]
...
-The cmdlet works by performing the following actions:
 
-- Deletes access rights for specified standard users to the root\cimv2 namespace on the local server (for access to role and feature inventory information).
+Disables access for one or more standard, non-Administrator users to event, service, performance
+counter, and role and feature inventory data for a server that you are managing by using Server
+Manager. The cmdlet restores the default, administrator-only access to this data, and must be run
+locally on the server that is being managed by using Server Manager. The cmdlet works by performing
+the following actions:
+
+- Deletes access rights for specified standard users to the root\cimv2 namespace on the local server
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md:33]:
```suggestion
- Deletes access rights for specified standard users to the `root\cimv2` namespace on the local server
```

File: docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md [hunk 3]
```diff
@@ -21,40 +23,57 @@ Disable-ServerManagerStandardUserRemoting [-User] <String[]> [-Force] [-WhatIf]
...
 ### Example 2
+
 ```powershell
 Disable-ServerManagerStandardUserRemoting -User JennyL -WhatIf
 ```
 
-In the following example, the administrator views the outcome of running a command to deny a standard user named JennyL access to event, performance counter, service status, and role and feature inventory data for a server that is being managed by using the Server Manager console running on either the local or a remote computer.
-The `WhatIf` parameter is added, meaning that the command actions are not carried out.
+In this example, the administrator views the outcome of running a command to deny a standard user
+named JennyL access to event, performance counter, service status, and role and feature inventory
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md:62]:
```suggestion
named `JennyL` access to event, performance counter, service status, and role and feature inventory
```

File: docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md [hunk 4]
```diff
@@ -21,40 +23,57 @@ Disable-ServerManagerStandardUserRemoting [-User] <String[]> [-Force] [-WhatIf]
...
 
 ### Example 3
+
 ```powershell
 Disable-ServerManagerStandardUserRemoting -User JennyL -Confirm
 ```
 
-In the following example, the administrator denies a standard user named JennyL access to event, performance counter, service status, and role and feature inventory data for a server that is being managed by using the Server Manager console running on either the local or a remote computer.
-The `Confirm` parameter is added, meaning that the command prompts for confirmation before performing the action.
+In this example, the administrator denies a standard user named JennyL access to event, performance
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md:73]:
```suggestion
In this example, the administrator denies a standard user named `JennyL` access to event, performance
```

File: docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md [hunk 5]
```diff
@@ -121,7 +145,11 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: `-Debug`, -`ErrorAction`, -`ErrorVariable`,
+-`InformationAction`, -`InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`,
+`-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see
+[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/ServerManager/Disable-ServerManagerStandardUserRemoting.md:152]: This is the rare moment where we _don't_ format parameters - this formatting gets overwritten when the cmdlet gets updated from PlatPS.

```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Thanks so much for this contribution, @raynbowbrite! A few small fixes, but we super appreciate your help!

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**